Configurations.md

```diff
@@ -2734,7 +2733,7 @@ fn lorem<Ipsum: Dolor+Sit=Amet>() {
 
 ## `unstable_features`
 
-Enable unstable features on the unstable channel.
+**Deperacted and ignored.** Formerly used to enable unstable features on the unstable channel. Unstable features are now activated regardless of whether this option is present or not.
```

Comment:
I did some digging and it turns out that the `--unstable_features` flag turns on whether or not you can pass `--skip-children`, `--error-on-unformatted` or `--file-lines` via the command line. [Here's the reference in the source code](https://github.com/rust-lang/rustfmt/blob/ee2bed96d60fd7e46b1fb868f6a8f27e3a8058d0/src/bin/main.rs#L544-L563)

Try running: `rustfmt +nightly --skip-children --error-on-unformatted --file-lines "[]"` and you'll get the following output:

```
Unstable options (`--skip-children`, `--error-on-unformatted`, `--file-lines`) used without `--unstable-features`
```

Also note the [`(unstable)` output](https://github.com/rust-lang/rustfmt/blob/ee2bed96d60fd7e46b1fb868f6a8f27e3a8058d0/src/bin/main.rs#L148-L172) for these options using of ` rustfmt +nightly --help` (emphasis with '*' added by me):

```
        --file-lines JSON
                        Format specified line ranges. Run with
                        `--help=file-lines` for more detail **(unstable)**.
        --error-on-unformatted 
                        Error if unable to get comments or string literals
                        within max_width, or they are left with trailing
                        whitespaces **(unstable)**.
        --skip-children 
                        Don't reformat child modules **(unstable)**.
```

Trying to run `rustfmt +stable --skip-children`, `rustfmt +stable --error-on-unformatted`, or `rustfmt +stable --file-lines "[]"`  results in an error because these options are [only added to the command line interface on nightly](https://github.com/rust-lang/rustfmt/blob/ee2bed96d60fd7e46b1fb868f6a8f27e3a8058d0/src/bin/main.rs#L148-L172) :

```
Unrecognized option: 'skip-children'
Unrecognized option: 'error-on-unformatted'
Unrecognized option: 'file-lines'
```